src/data-access/repositories/mongo/mongo-ingested-segment-changed-listener.ts

```diff
@@ -42,7 +42,7 @@ export class MongoIngestedSegmentChangedListener extends BaseMongoRepository imp
       case MongoChangeEvent.INSERT: {
         const insertChange: ChangeStreamInsertDocument<MongoIngestedSegment> = change as ChangeStreamInsertDocument<MongoIngestedSegment>
         const ingestedSegmentId: string = insertChange.fullDocument._id
-        const ingestedSegment: IngestedSegment = await this.ingestedSegmentRepository.getIngestedSegment(ingestedSegmentId)
+        const ingestedSegment: IngestedSegment = await this.ingestedSegmentRepository.getIngestedSegmentRundown(ingestedSegmentId)
```

Comment:
What is a Ingested Segment Rundown? 😁

Reply:
A spelling mistake :D 
It's been updated